### DISCUSSION TITLE
Unable to set values Aeotec Wallmote

### DISCUSSION THREAD

**Discussion by Robbie1983** [zwave-js/zwave-js#3889] [Q&A]
Hi everyone, 

I am slowly moving from wired (self made mqtt based domitica) to zwave based commercial switches as the wife wants a more stylish looking switch assembly. However, I am struggling a little with the configuration of the different switches. For example a aeotec wallmote. 

The wallmote is succesfully added and working, sending notifications and also the values are accessible for reading.
However, I am unable to set the configuration parameters both through CCapi or ValueApi.

For example the  touch sound should be adjustable trough the "configuration" parameter=1.

![image](https://user-images.githubusercontent.com/95997962/145704433-db17a218-fa32-4213-947e-a4d6bcb1ca21.png)

however sending:

let Message = {
 payload: {
 mode: "CCAPI",
 node: 5,
 cc: "Configuration",
 method: "set",
 params: [1,0,1]
 }
}
return Message

Will not set the touch sound to disable. Also the other configuration parameters seem not to be editable. 
Any help is appreciated......

**Comment by marcus-j-davies** [comment]
@AlCalzone,
This is being discussed over at my repo - I got it (its a battery operated device - so is likely asleep)

**Reply by AlCalzone** [reply]
> its a battery operated device - so is likely asleep

That's most likely it. The device must be woken up with the button on the back for config changes to go through.
